ReactNativeClient/lib/components/screens/config.js

```diff
@@ -250,6 +250,8 @@ class ConfigScreenComponent extends BaseScreenComponent {
 		// styles.switchSettingControl.width = '20%';
 		styles.switchSettingControl.flex = 0;
 
+		styles.keyboardAppearance = theme.keyboardAppearance;
```

Comment:
You don't need to add it there as you can simply get the theme where you need it using `const theme = themeStyle(this.props.theme);`